.github/workflows/main.yml

```diff
@@ -158,7 +158,7 @@ jobs:
         with:
           path: /tmp/luarocks
           # The build input for luarocks changes per test, so we need separate caches
-          key: ${{ runner.os }}-${{ matrix.test_name }}-luarocks-3.5.0
+          key: ${{ github.workflow }}-${{ runner.os }}-${{ matrix.test_name }}-luarocks-3.5.0
```

Comment:
Why can't we use the same `luarocks` cache ?

Reply:
We could, assuming that it handles the case correctly where both workflows run into the same cache miss at the same time.
But do we need to? If we have at least one CI run every 7 days, this cache will never invalidate. And even if it does, builds times for LuaRocks aren't bad (don't remember the number, but should be less than 2 minutes).